README.md

```diff
@@ -1,49 +1,105 @@
+
+## plenty system setup
+
+To run this project, you need a plenty system. Get a free 30-day trial [on our website](https://www.plentymarkets.com/) or choose one of the [available editions](https://www.plentymarkets.com/product/pricing/).
+
+### Plugins
+
+> :bulb: New systems ship with the required plugins pre-installed. If you use a new system, skip to the next step.
+
+For the shop to run, you have to install plugins that provide additional REST routes and providers. Carry out the following steps:
+
+1. Log into your plenty system.
+2. [Install](https://knowledge.plentymarkets.com/en-gb/manual/main/plugins/installing-added-plugins.html#installing-plugins) the following plugins in the latest version:
+    - IO
+    - plentyShop LTS
+    - PayPal
+    - Cash in advance
+    - Pay upon pickup
+    - DHL Shipping (Versenden)
+    - Customer feedback
+3. Set the priorities of the plugins, so that IO has the highest priority and plentyShop LTS the second highest.
 
-To get started, see the following guides:
+### System configuration
 
-- [Introduction](https://docs.vuestorefront.io/v2/getting-started/introduction.html) to learn what is Vue Storefront.
+1. Log into your plenty system.
+2. Go to **Setup » Guided Tours** and complete **Setting up plentyShop**.
+3. *Optional:* Go to **Setup » Orders » Payment » PayPal** and set up your PayPal account. If you skip this step, remove the PayPal component from the app.
 
-## Prerequisites:
+## Local dev setup
+
+This section describes how to run the project locally.
+
+### Prerequisites
 
 - `Node.js` 20+
-- `Yarn`
+- `Yarn` 3
 
-### Local dev setup
+Download Node.js from the [official website](https://nodejs.org/) or use [nvm](https://github.com/nvm-sh/nvm) to switch to a compatible version. For installation instructions for Yarn, refer to the [Yarn documentation](https://yarnpkg.com/getting-started/install).
 
-1. Clone the repository.
+### Repository fork
 
-```bash
-$ git clone git@github.com:vuestorefront/storefront-nuxt3-boilerplate.git
-```
+We recommend working with a fork of this repository. A fork allows you to easily incorporate updates from this boilerplate into your own codebase.
 
-2. Install all dependencies (read here [How to install Yarn 3](https://yarnpkg.com/getting-started/install)).
+1. [Create a fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
+2. [Clone the forked repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository).
 
-```bash
-$ yarn install
-```
+### Configuration
+
+- Open [`middleware.config.ts`](./apps/server/middleware.config.ts) and replace `url` with the URL of your plenty system.
+- Open [`i18n.config.ts`](./apps/web/i18n.config.ts) and [`nuxt.config.ts`](./apps/web/nuxt.config.ts) to update the locales. The locale settings have to match the plenty system settings. The `fallbackLocale` and `defaultLocale` have to match the system default language.
```

Comment:
"The locale settings have to match the plenty system settings." Isn't this redundant because the next sentence describes the same but in detail?

Reply:
I don't think it's redundant. The first sentence talks about locales in general, the other about the defaults.